src/drivers/default.rs

```diff
@@ -1,30 +1,62 @@
-pub struct Driver {
+use std::result;
+use std::error::Error;
+use std::fmt;
+use driver::Driver;
+
+type Result<T> = result::Result<T, NoError>;
+
+pub struct DefaultDriver {
   target: String,
-  status: isize,
+  status: isize
 }
 
-impl Driver {
+impl DefaultDriver {
```

Comment:
I still prefer `Driver` here since it would be called as `drivers::default::Driver` instead of `drivers::default::DefaultDriver` kinda repetitive there...